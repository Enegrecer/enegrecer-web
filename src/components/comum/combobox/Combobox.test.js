import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Combo from './index';

describe('Combobox', () => {
  const dados = ['Item 1', 'Item 2', 'Item 3', 'Item 4'];
  it('nÃ£o deve possuir estrutura definida ', () => {
    const onChange = jest.fn();
    const combo = mount(<Combo itens={dados} onChange={onChange} />);
    const comboTextoJson = toJson(combo);
    expect(comboTextoJson).toMatchSnapshot();
  });
});

