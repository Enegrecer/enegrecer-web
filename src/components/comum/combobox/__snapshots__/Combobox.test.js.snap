// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Combobox deve possuir estrutura definida  1`] = `
<div>
  <select
    name="select"
  >
    <option
      disabled={true}
      selected={true}
      value=""
    />
    <option
      key="Item 1"
      value="Item 1"
    >
      Item 1
    </option>
    <option
      key="Item 2"
      value="Item 2"
    >
      Item 2
    </option>
    <option
      key="Item 3"
      value="Item 3"
    >
      Item 3
    </option>
    <option
      key="Item 4"
      value="Item 4"
    >
      Item 4
    </option>
  </select>
</div>
`;
